radixdlt-core/radixdlt/src/main/java/com/radixdlt/api/core/CoreApiModule.java

```diff
@@ -152,5 +154,9 @@ public void configure() {
       routeBinder.addBinding(HandlerRoute.post("/key/vote")).to(VoteHandler.class);
       routeBinder.addBinding(HandlerRoute.post("/key/withdraw-vote")).to(WithdrawVoteHandler.class);
     }
+
+    if (olympiaStateExportEnabled) {
+      routeBinder.addBinding(HandlerRoute.get("/olympia-state")).to(OlympiaStateHandler.class);
```

Comment:
Disabled by default because this is rather unsafe in the current form and only really meant for testing

Reply:
Could elaborate more about only meant for testing. I've seen few examples of code only meant for testing inside classes and I don't believe it is a good idea.

Reply:
I think it makes sense to not include this API on `main`, I've kept it on a separate branch: `feature/state-prop-for-babylon-testing-api` and removed from this PR.

Reply:
Ahh nice, sounds good :)